scripts/verify.bash

```diff
@@ -72,7 +72,7 @@ go vet \
     $DIRNAMES || [ -n "$IGNORE_VET_WARNINGS" ]
 
 # Allow the ignoring of the golinters
-if [ -z "$IGNORE_GOLINTERS" ]; then
+if [ -n "$IGNORE_GOLINTERS" ]; then
```

Comment:
I think we have to change the env var. Having "IGNORE_GOLINTERS=1" mean *don't* ignore, is confusing.
RUN_GOLINTERS
INCLUDE_GOLINTERS
DONT_IGNORE_GOLINTERS
I think I like INCLUDE the most.

Reply:
💯 agree, naming is hard and this makes sense